jsonschema-validation.xml

```diff
@@ -539,26 +554,60 @@
                 <t>
-                    The "format" keyword functions as both an annotation
-                    and as an assertion.  While no special effort is required to
-                    implement it as an annotation conveying semantic meaning, implementing
-                    validation is non-trivial.
-                </t>
-                <t>
-                    Implementations MAY support the "format" keyword as a validation assertion.
-                    Should they choose to do so:
-
+                    The "format" keyword functions as an annotation, and optionally as an assertion.
+                    Declaring the format vocabulary in "$vocabulary" with a value of true indicates
+                    that an implementation MUST treat the keyword as an assertion.
+                    This means that:
                     <list>
                         <t>they SHOULD implement validation for attributes defined below;</t>
                         <t>they SHOULD offer an option to disable validation for this keyword.</t>
                     </list>
+                </t>
 
+                <t>
+                    Due to the complexity involved in fully validating some format attributes
+                    defined in this specification, implementations MAY provide only limited
+                    validation support for some format attributes.  Implementations SHOULD
+                    document any such intentional limitations.
                 </t>
 
                 <t>
-                    Implementations MAY add custom format attributes. Save for agreement between
-                    parties, schema authors SHALL NOT expect a peer implementation to support this
-                    keyword and/or custom format attributes.
+                    The <xref target="meta-schema">standard core and validation meta-schema</xref>
+                    includes this vocabulary in its "$vocabulary" keyword with a value of false,
+                    since by default implementations are not required to support this keyword
+                    as an assertion.
+                </t>
+
+                <t>
+                    When the format vocabulary is declared with a value of false, an implementation
+                    SHOULD treat "format" as an annotation keyword, to facilitate applications which
+                    wish to do their own semantic validation.
+                    <cref>
+                        This is not the normal behavior for a vocabulary declared with false, which is
+                        why the requirement is a SHOULD.  Implementations MAY ignore "format" entirely
+                        as is allowed by false vocabulary declarations.  However, due to the long history
+                        of this keyword, treating it as something of a special case seems reasonable.
+                        This may be revised in future drafts based on feedback.
```

Comment:
There's been some discussion around "should all unknown keywords be collected as annotations for application use?". Doing so would prevent the need to define this explicit behaviour, which as you note, is different to when other vocabs are given a false value.

Reply:
Yeah I decided to separate that conversation from this one.  This will work fine now, and does not prevent us from making annotation collection the default behavior later.  Either before or after publishing this draft, but we really do need to wrap this up soon, and changing the overall default behavior may be pretty contentious.

Clarifying `format` is fine to do because it is such an enormous source of complaints, and none of this is incompatible with existing behavior.

Reply:
Overall I think this is fine.
However, it does lead me to consider that this is placing an extra implementation SHOULD based on a vocab being false. Do you think it's worth saying something like " generally it is not recomend that behaviour is derived from a vocabulary having a false value"? or do you think that's overkill?

Reply:
This is covered more clearly in the new text I had written up on Sunday.  I'm just finishing up the minimum validation requirements now and will post an update by tonight.